packages/react/src/components/MadeBy/MadeBy.tsx

```diff
@@ -0,0 +1,96 @@
+import { MadeByProps, MadeByTypeMap } from './MadeBy.types';
+
+import clsx from 'clsx';
+import { getMadeByUtilityClass, madeByClasses } from './MadeBy.classes';
+
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+
+import { useThemeProps } from '@mui/material/styles';
+import { styled } from '@mui/material';
+import { OverridableComponent } from '@mui/material/OverridableComponent';
+
+import { IconElonsoft } from '../../icons';
+import { svgIconClasses } from '../SvgIcon';
+
+type MadeByOwnerState = {
+  classes?: MadeByProps['classes'];
+  clickable?: MadeByProps['clickable'];
+};
+
+const useUtilityClasses = (ownerState: MadeByOwnerState) => {
+  const { classes, clickable } = ownerState;
+
+  const slots = {
+    root: ['root', clickable && 'clickable']
+  };
+
+  return composeClasses(slots, getMadeByUtilityClass, classes);
+};
+
+const MadeByRoot = styled('div', {
+  name: 'ESBadge',
+  slot: 'Root',
+  overridesResolver: (props, styles) => {
+    const {
+      ownerState: { clickable }
+    } = props;
+
+    return [styles.root, clickable && styles.clickable];
+  }
+})(({ theme }) => ({
+  ...theme.typography.caption,
+  color: theme.palette.monoA.A500,
+  cursor: 'default',
+  display: 'flex',
+  gap: '2px',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  width: 'fit-content',
+
+  [`& .${svgIconClasses.root}`]: {
+    color: theme.palette.monoA.A300
+  },
+
+  [`&.${madeByClasses.clickable}`]: {
+    cursor: 'pointer',
+
+    [`&:hover .${svgIconClasses.root}`]: {
+      color: theme.palette.monoA.A400
+    },
+    [`&:focus .${svgIconClasses.root}`]: {
```

Comment:
Посмотрел в фигме, всё же это
```suggestion
    [`&:active .${svgIconClasses.root}`]: {
```